apps/spruce/src/components/Icon/icons/FallTree.tsx

```diff
@@ -1,105 +1,126 @@
-            fill="freeze"
-            keyTimes="0;0.1;0.2;0.3;0.4;0.5;0.6;0.7;0.8;0.9;1"
-            repeatCount="indefinite"
-            values="0.0197893;0.0222233;0.0222233;0.199525;0.496432;0.745;0.496432;0.199525;0.0222233;0.0222233;0.0197893"
-          />
-          <animate
-            attributeName="stop-color"
-            begin="0s"
-            calcMode="linear"
-            dur="13s"
-            fill="freeze"
-            keyTimes="0;0.1;0.2;0.3;0.4;0.5;0.6;0.7;0.8;0.9;1"
-            repeatCount="indefinite"
-            values="#EBB906;#E4150E;#E4150E;#E4150E;#E4150E;#E4150E;#E4150E;#E4150E;#E4150E;#E4150E;#EBB906"
-          />
-        </stop>
-        <stop offset=".022" stopColor="#E4150E">
-          <animate
-            attributeName="offset"
-            begin="0s"
-            calcMode="linear"
-            dur="13s"
-            fill="freeze"
-            keyTimes="0;0.1;0.2;0.3;0.4;0.5;0.6;0.7;0.8;0.9;1"
-            repeatCount="indefinite"
-            values="0.0222233;0.0437326;0.315184;0.522266;0.69419;0.835;0.69419;0.522266;0.315184;0.0437326;0.0222233"
-          />
-          <animate
-            attributeName="stop-color"
-            begin="0s"
-            calcMode="linear"
-            dur="13s"
-            fill="freeze"
-            keyTimes="0;0.1;0.2;0.3;0.4;0.5;0.6;0.7;0.8;0.9;1"
-            repeatCount="indefinite"
-            values="#E4150E;#E86C0A;#E86C0A;#E86C0A;#E86C0A;#EBB906;#E86C0A;#E86C0A;#E86C0A;#E86C0A;#E4150E"
-          />
-        </stop>
-        <stop offset=".044" stopColor="#E86C0A">
-          <animate
-            attributeName="offset"
-            begin="0s"
-            calcMode="linear"
-            dur="13s"
-            fill="freeze"
-            keyTimes="0;0.1;0.2;0.3;0.4;0.5;0.6;0.7;0.8;0.9;1"
-            repeatCount="indefinite"
-            values="0.0437326;0.437506;0.693037;0.780395;0.780395;0.88;0.780395;0.780395;0.693037;0.437506;0.0437326"
-          />
-          <animate
-            attributeName="stop-color"
-            begin="0s"
-            calcMode="linear"
-            dur="13s"
-            fill="freeze"
-            keyTimes="0;0.1;0.2;0.3;0.4;0.5;0.6;0.7;0.8;0.9;1"
-            repeatCount="indefinite"
-            values="#E86C0A;#EBB906;#EBB906;#EBB906;#EBB906;#E86C0A;#EBB906;#EBB906;#EBB906;#EBB906;#E86C0A"
-          />
-        </stop>
-        <stop offset=".583" stopColor="#00A35C">
-          <animate
-            attributeName="offset"
-            begin="0s"
-            calcMode="linear"
-            dur="13s"
-            fill="freeze"
-            keyTimes="0;0.1;0.2;0.3;0.4;0.5;0.6;0.7;0.8;0.9;1"
-            repeatCount="indefinite"
-            values="0.582776;0.838822;0.900502;0.900502;0.900502;0.900502;0.900502;0.900502;0.900502;0.838822;0.582776"
-          />
-        </stop>
-      </linearGradient>
-    </defs>
-  </svg>
-);
+import { useEffect, useRef } from "react";
+
+export const FallTree: React.FC = () => {
```

Comment:
Might we extract this into a reusable "pause icon" component that imports the fall tree? This seems like a pattern we ought to keep around